.github/workflows/gh-pages.yml

```diff
@@ -17,6 +17,9 @@ jobs:
       - name: Dependencies
         run: |
           if [ -f pyproject.toml ]; then pip install ".[docs]"; fi
+      - name: Copy the notebooks
```

Comment:
I'd favor a symlink. My fault, but I couldn't find how to pass `conf.py` to `sphinx-build`. I'd favor keeping documentation config under `/docs`, instead of `/`.

Reply:
Yeah, by putting this in the CI build and ignoring the notebooks under `/docs` in the `.gitignore` file was my workaround (and let us keep documentation under `/docs` and the example notebooks under `/Notebooks`), but if the symlink works that'd be nicer.